workflows/check_commit_message_for_tickets.yml

```diff
@@ -0,0 +1,92 @@
+name: Check Commit Message
+
+on:
+  pull_request:
+    types: [opened, synchronize, edited, reopened]
+
+jobs:
+  check-commit-message:
+    runs-on: ubuntu-latest
+    env:
+      REGEX_PATTERN: '^(?i)(ref|refs|reference|references|res|resolve|resolves)[ \t]*:[ \t]*(gh-|\#|\!|[A-Za-z]+-)\d+\s*$'
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

Comment:
Could pin to a git commit [see](https://github.com/enterprise-contract/.github/blob/dcef45175ab0493370c5eb85f9a08ffbbf21d75b/.github/workflows/scorecards.yml#L39)

Reply:
Thank you. I'll do this.